components/lv-ts-tableprovider/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+ 
+  <groupId>com.tibco.community</groupId>
+  <artifactId>ts-table-demo</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+ 

```

Comment:
This is the place where the name and description for "this" sample should exist (what will be read by Studio to show the name and description)

Reply:
Ah, I put name/description in the two sub-projects. I should remove both those and put one at the top, yes?